pkg/minikube/problem/err_map.go

```diff
@@ -61,6 +61,11 @@ var vmProblems = map[string]match{
 		Advice: "Disable Hyper-V when you want to run VirtualBox to boot the VM",
 		Issues: []int{4051},
 	},
+	"HYPERV_NO_VSWITCH": {
+		Regexp: re(`no External vswitch found. A valid vswitch must be available for this command to run.`),
+		Advice: "Hyper-V requires an external network switch to function properly.",
```

Comment:
The advice here is intended to be worded as something the user should perform to work around this. Like perhaps: "Configure an external network switch using x-y command"

Reply:
Done.